Casks/f/foks.rb

```diff
@@ -17,7 +17,8 @@
     end
   end
 
-  binary "foks"
+  binary "foks", target: "foks"
```

Comment:
```suggestion
  binary "foks"
```

Using `target:` should not be required in this case.